src/compiler/displayOutput.ml

```diff
@@ -243,7 +243,13 @@ let handle_display_argument com file_pos pre_compilation did_something =
 				DMDefinition
 			| "usage" ->
 				Common.define com Define.NoCOpt;
-				DMUsage false
+				DMUsage (false,false,false)
+			| "usage@descendants" ->
```

Comment:
I wonder if we should still add support for new features to the legacy protocol?

Reply:
It's used for our display tests.

Reply:
Sure, but I'm not sure it should be. :)